packages/create-sitecore-jss/src/templates/nextjs/src/components/SafeHydrate.tsx

```diff
@@ -0,0 +1,9 @@
+﻿import dynamic from 'next/dynamic';
+import React from 'react';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const NonSSRWrapper = (props: any) => <React.Fragment>{props.children}</React.Fragment>;
+
+export default dynamic(() => Promise.resolve(NonSSRWrapper), {
```

Comment:
Would be nice if you can add some description to the component, in order to understand why it was added

Reply:
I have added some description where I am using the component i.e. <Layout />. Where do you think it's more appropriate?

Reply:
@sc-addypathania I mean above the `NonSSRWrapper` component definition, basically in both places it worth to mention, I think